monai/transforms/utils.py

```diff
@@ -1171,6 +1176,247 @@ def get_largest_connected_component_mask(
 
     return convert_to_dst_type(out, dst=img, dtype=out.dtype)[0]
 
+def get_largest_connected_component_mask_point(
+    img_pos: NdarrayTensor,
+    img_neg: NdarrayTensor,
+    pos_val: list=[1, 3],
+    neg_val: list=[0, 2],
+    point_coords: None = None,
+    point_labels: None = None,
+    margins: int = 3,
+) -> NdarrayTensor:
+    """
+    Gets the largest connected component mask of an image that include the point_coords.
+    Args:
+        img_pos: [1, B, H, W, D]
+        point_coords [B, N, 3]
+        point_labels [B, N]
+    """
+
+    img_pos_, *_ = convert_data_type(img_pos, np.ndarray)
+    img_neg_, *_ = convert_data_type(img_neg, np.ndarray)
+    label = measure.label
+    lib = np
+
+    features_pos, num_features = label(img_pos_, connectivity=3, return_num=True)
+    features_neg, num_features = label(img_neg_, connectivity=3, return_num=True)
+
+    outs = np.zeros_like(img_pos_)
+    for bs in range(point_coords.shape[0]):
+        for i, p in enumerate(point_coords[bs]):
+            if point_labels[bs, i] in pos_val:
+                features = features_pos
+            elif point_labels[bs, i] in neg_val:
+                features = features_neg
+            else:
+                # if -1 padding point, skip
+                continue
+            for margin in range(margins):
+                left, right = max(p[0].round().int().item() - margin, 0), min(
+                    p[0].round().int().item() + margin + 1, features.shape[-3]
+                )
+                t, d = max(p[1].round().int().item() - margin, 0), min(
+                    p[1].round().int().item() + margin + 1, features.shape[-2]
+                )
+                f, b = max(p[2].round().int().item() - margin, 0), min(
+                    p[2].round().int().item() + margin + 1, features.shape[-1]
+                )
```

Comment:
```suggestion
                x, y, z = p.round().int().tolist()
                left, right = max(x - margin, 0), min(x + margin + 1, features.shape[-3])
                t, d = max(y - margin, 0), min(y + margin + 1, features.shape[-2])
                f, b = max(z - margin, 0), min(z + margin + 1, features.shape[-1])
```
Something like this is more readable?